ilastik/applets/networkClassification/opNNclass.py

```diff
@@ -340,6 +343,30 @@ def mergeLabels(self, from_label, into_label):
     def clearLabel(self, label_value):
         for laneIndex in range(len(self.InputImages)):
             self.getLane(laneIndex).opLabelPipeline.opLabelArray.clearLabel(label_value)
+
+    def get_val_layer(self, parameters):
```

Comment:
* Functions named `get*` usually return something, but this function assigns and doesn't return anything.
* `val` abbreviation usually means "value" and probably should be expanded here.

```suggestion
    def set_validation_mask(self, parameters):
```